config/express.js

```diff
@@ -1,49 +1,51 @@
 var passport = require('passport'),
   RedditStrategy = require('passport-reddit').Strategy;
 
-var verifyHandler = function(token, tokenSecret, profile, done) {
-  process.nextTick(function() {
-    User.findOne({uid: profile.id}, function(err, user) {
-      Reddit.getBothFlairs(tokenSecret, profile.name, function (flair1, flair2) {
-        if (user) {
-          if (user.banned) {
-            return done("You are banned from FAPP", user);
-          }
-          user.redToken = tokenSecret;
-          user.flair = {ptrades: flair1, svex: flair2};
-          user.save(function (err) {
-            if (!err) {
-              return done(null, user);
-            } else {
-              return done(null, user);
+module.exports = {
+  verifyHandler: function(token, tokenSecret, profile, done) {
```

Comment:
The getFlair() function here has been failing ever since we stopped asking users for `flair` scope. However, we never noticed it due to buggy error handling.

Now it uses the `adminRefreshToken` to get the users flairs, as it's supposed to. Since `sails.config.adminRefreshToken` is a global sails variable, the entire verifyHandler() function needs to be in `module.exports` in order to access it.